### PR TITLE
Clarify OUs in CA Certificates

### PR DESCRIPTION
**Pull request by dzacharo**
Clarify that OUs are not allowed for "CA Certificates described in section 7.1.2.3" per conversation in the [validation subcommittee mailing list](https://lists.cabforum.org/pipermail/validation/2022-October/001812.html).

Added an effective date of 2022-12-12 which can be updated as needed.

### CODE REVIEW

File: docs/BR.md [hunk 1]
```diff
@@ -2691,6 +2691,7 @@ The following table details the acceptable `AttributeType`s that may appear with
...
 | `streetAddress`          | MAY             | If present, the CA's street address. Multiple instances MAY be present. | [Section 3.2.2.1](#3221-identity) |
 | `organizationName`       | MUST            | The CA's name or DBA. The CA MAY include information in this field that differs slightly from the verified name, such as common variations or abbreviations, provided that the CA documents the difference and any abbreviations used are locally accepted abbreviations; e.g. if the official record shows "Company Name Incorporated", the CA MAY use "Company Name Inc." or "Company Name". | [Section 3.2.2.2](#3222-dbatradename) |
+| `organizationalUnitName`  | _Effective 2022-12-12_ "MUST NOT" for Root CA Certificates defined in [Section 7.1.2.1](#7121-root-ca-certificate-profile) or TLS Subordinate CA Certificates defined in [Section 7.1.2.5](#7125-technically-constrained-tls-subordinate-ca-certificate-profile) or Technically-Constrained TLS Subordinate CA Certificate defined in [Section 7.1.2.6](#7126-tls-subordinate-ca-certificate-profile). "SHOULD NOT" for all other CAs | -           | -           |
```
Comment (ryancdickson) [inline docs/BR.md:2694]: Is this more clear (no worries, if it's not!)?

| `organizationalUnitName`       | -               | -           | -           |
| \ \ \ \ _Effective 2022-12-12_ for Root CA Certificates defined in [Section 7.1.2.1](#7121-root-ca-certificate-profile), TLS Subordinate CA Certificates defined in [Section 7.1.2.5](#7125-technically-constrained-tls-subordinate-ca-certificate-profile), or Technically-Constrained TLS Subordinate CA Certificates defined in [Section 7.1.2.6](#7126-tls-subordinate-ca-certificate-profile) | MUST NOT |  -  |
| \ \ \ \ _Effective 2022-12-12_ for all other CAs | SHOULD NOT | - |

We could also consider adding a third line to cover the "prior" period (i.e., "Prior to 2022-12-12").

Comment (XolphinMartijn) [reply docs/BR.md:2694]: I'll also add that we probably should change "CAs" to "CA Certificates"

Comment (CBonnell) [reply docs/BR.md:2694]: We have a top-level effective date for this ballot of 2023-04-15, so I don't think we need another effective date here.

Comment (CBonnell) [inline docs/BR.md:2694]:
```suggestion
| `organizationalUnitName`  | This attribute MUST NOT be included in Root CA Certificates defined in [Section 7.1.2.1](#7121-root-ca-certificate-profile) or TLS Subordinate CA Certificates defined in [Section 7.1.2.5](#7125-technically-constrained-tls-subordinate-ca-certificate-profile) or Technically-Constrained TLS Subordinate CA Certificate defined in [Section 7.1.2.6](#7126-tls-subordinate-ca-certificate-profile). For all other types of CA Certificate, this attribute SHOULD NOT be included. | -           | -           |
```

File: docs/BR.md [hunk 2]
```diff
@@ -2691,6 +2691,7 @@ The following table details the acceptable `AttributeType`s that may appear with
...
 | `streetAddress`          | MAY             | If present, the CA's street address. Multiple instances MAY be present. | [Section 3.2.2.1](#3221-identity) |
 | `organizationName`       | MUST            | The CA's name or DBA. The CA MAY include information in this field that differs slightly from the verified name, such as common variations or abbreviations, provided that the CA documents the difference and any abbreviations used are locally accepted abbreviations; e.g. if the official record shows "Company Name Incorporated", the CA MAY use "Company Name Inc." or "Company Name". | [Section 3.2.2.2](#3222-dbatradename) |
+| `organizationalUnitName`  | This attribute MUST NOT be included in Root CA Certificates defined in [Section 7.1.2.1](#7121-root-ca-certificate-profile) or TLS Subordinate CA Certificates defined in [Section 7.1.2.5](#7125-technically-constrained-tls-subordinate-ca-certificate-profile) or Technically-Constrained TLS Subordinate CA Certificate defined in [Section 7.1.2.6](#7126-tls-subordinate-ca-certificate-profile). This attribute SHOULD NOT be included in other types of CA Certificates. | -           | -           |
```
Comment (ryancdickson) [inline docs/BR.md:2694]: Minor typo: missing an "s" after "or Technically-Constrained TLS Subordinate CA Certificate"

This attribute MUST NOT be included in Root CA Certificates defined in [Section 7.1.2.1](#7121-root-ca-certificate-profile) or TLS Subordinate CA Certificates defined in [Section 7.1.2.5](#7125-technically-constrained-tls-subordinate-ca-certificate-profile) or Technically-Constrained TLS Subordinate CA Certificate**s** defined in [Section 7.1.2.6](#7126-tls-subordinate-ca-certificate-profile). This attribute SHOULD NOT be included in other types of CA Certificates.

Didn't want to disrupt the call by bringing this up.

**Pull request merged**